platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/fontscolors/FlatLafDark-INI-tokenColorings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing the license header.

Reply:
Right, missed it!

Will fix it this evening.

Reply:
Also this one does not contains color definitions, so it could be named as FlatLaf-INI-tokenColorings.xml and can be registered bot for FlatLafDark and FlatLafLight Look and Feels.

Reply:
Added license.

I would prefer to deal with FlatLaf Light at a later step.